parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -161,6 +161,9 @@ else
     time_metric "RemoveEtcd" removeEtcd
 fi
 
+{{/* configure /etc/ssh/sshd_config on master if the ssh alternative port is enabled */}}
+time_metric "ConfigureMasterSSHD" configureMasterSSHD
```

Comment:
should this only be run on master nodes?

Reply:
I've made a check before writing to sshd_config.